packages/hooks/src/Provider.tsx

```diff
@@ -46,15 +48,50 @@ export const Provider: React.FC<ProviderProps> = ({ children, network, infuraId
   const [connected, setConnected] = React.useState<boolean>(false);
 
   const connectWallet = React.useCallback(async () => {
+    // Coinbase walletLink init
+    const walletLink = new WalletLink({
+      appName: 'coinbase',
+    });
+
+    // WalletLink provider
+    const walletLinkProvider = walletLink.makeWeb3Provider(
+      'https://eth-mainnet.alchemyapi.io/v2/qCdzfF9UqXcJYIle-Ff-BN0MII8LjLQs',
```

Comment:
i see this url used in 2 places, let's pull it out into a constant?

Reply:
Did you all really want to expose that endpoint out in the wild?  Theoretically anyone can steal that url and use it for themselves.  I would suggest perhaps hiding it in an env variable is possible or let the consumer pass in there own rpc link

Reply:
Yes, I agree, we need to accept this as a prop from the user. I'm using it here for development purposes. 
It's not even mine lol, I got that from one from the repo I used as a ref.

Reply:
Cool, cool yeah these rpc links seem to magically appear in all the projects lol

Reply:
I think it makes if we could define and agree on some defaults. It makes sense to define URLs and constants in a common file if they can be exposed. In my opinion it makes the code more beginner friendly and easier to read. 

The second argument for the `makeWeb3Provider` is the chainId, I think a it's hard for a new user to understand waht it is and to imply that he knows that the chainID 1 is for ETH mainnet.